guides/common/modules/snip_do-not-use-both-ldap-and-freeipa.adoc

```diff
@@ -0,0 +1,4 @@
+Users cannot use both {FreeIPA} and LDAP as an authentication method.
+Once a user authenticates using one method, that user cannot use the other method.
```

Comment:
```suggestion
Once a user authenticates using one method, they cannot use the other method.
```

Reply:
I know the IBM style guide allows for us to use `they` from their list of approved pronouns. Would it sound better to use that here instead of `that user`?

Reply:
Sure, thanks for the tip! Applied the suggestion and ready for re-review.